.travis/tests.sh

```diff
@@ -10,14 +10,20 @@ echo "" > coverage.txt
 # tests.
 ip a
 
+GO_TEST_OPTS=()
+if [[ "$TRAVIS_GO_VERSION" =~ ^1.(9|10|11|12)$ ]]
+then
+    GO_TEST_OPTS+='-vet=off'
```

Comment:
Maybe add a comment that this is because we are using the `%w` verb.

Reply:
Fixed.

Reply:
Fixed.